pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -2117,24 +2171,27 @@ Invitation:
       description: the id of the invitation
       type: string
     createdAt:
-      description: timestamp of when invitation was created, in ISO-8601 UTC format
+      description: timestamp of when invitation was created.
       type: string
       example: 2020-05-13T17:58:33.887Z
+      format: YYYY-MM-DDThhmmss.sssZ (ISO-8601)
     voidedAt:
-      description: timestamp of when invitation was voided, making the invitation invalid, in ISO-8601 UTC format
+      description: timestamp of when invitation was voided, making the invitation invalid.
       type: string
       nullable: true
       example: null
     verifiedAt:
-      description: timestamp of when invitation was verified, indicating user has acknowledged receiving the invitation, in ISO-8601 UTC format
+      description: timestamp of when invitation was verified, indicating user has acknowledged receiving the invitation.
       type: string
       nullable: true
       example: 2020-05-13T18:58:33.887Z
+      format: YYYY-MM-DDThhmmss.sssZ (ISO-8601)
     acceptedAt:
-      description: timestamp of when invitation was accepted, in ISO-8601 UTC format
+      description: timestamp of when invitation was accepted.
       type: string
       nullable: true
       example: 2020-05-13T19:58:33.887Z
+      format: YYYY-MM-DDThhmmss.sssZ (ISO-8601)
```

Comment:
Do we need colon separator in the time portion of the format?

Reply:
Ooo, good catch. Yes, we do- we're using the extended format, not the basic. Thank you!

Reply:
Fixed in the latest commit.